Dialog/PatchEditDialog.cpp

```diff
@@ -0,0 +1,117 @@
+#include "PatchEditDialog.h"
+#include "ui_PatchEditDialog.h"
+#include <QFileDialog>
+#include <ROMUtils.h>
+
+/// <summary>
+/// Construct an instance of the PatchEditDialog.
+/// </summary>
+/// <param name="parent">
+/// The parent QWidget.
+/// </param>
+/// <param name="patchEntry">
+/// The patch entry whose fields are used to initialize the dialog.
+/// </param>
+PatchEditDialog::PatchEditDialog(QWidget *parent, struct PatchEntryItem patchEntry) :
+    QDialog(parent),
+    ui(new Ui::PatchEditDialog)
+{
+    ui->setupUi(this);
+
+    // Initialize the items in comboBox_PatchType
+    QStringList PatchTypeNameSet;
+    PatchTypeNameSet << "Binary" << "Assembly" << "C";
```

Comment:
don't we need to initialize them to be static in the class?

Reply:
Not **needed**, but preferred. Good catch, I will fix this

Reply:
Fixed